tests/test_main.py

```diff
@@ -79,6 +88,30 @@ def test_construct_essence_sidecar():
     assert sidecar_xml.decode("utf-8") == MOCK_MEDIAHAVEN_EXTERNAL_METADATA
 
 
+def test_construct_collateral_sidecar():
+    # ARRANGE
+    event = json.loads(S3_MOCK_COLLATERAL_EVENT)
+
+    # ACT
+    sidecar_xml = construct_collateral_sidecar(event,
+                                               "test_pid",
+                                               "media_id",
+                                               "VRT",
+                                               "metadata"
+                                               )
```

Comment:
Formatted a bit more clearly:

```suggestion
    sidecar_xml = construct_collateral_sidecar(
        event, "test_pid", "media_id", "VRT", "metadata"
    )
```

Reply:
Agree, but my formatter gave a warning when I tried this